src/style.scss

```diff
@@ -198,8 +201,34 @@ $picker-disabled-background: white !default;
     border: 1px solid $picker-border;
     border-radius: ($picker-border-radius + 1);
     width: $picker-width;
+    position: relative;
   }
 
+  .react-datepicker-container:after,
```

Comment:
this is SASS, you can incapsulate selectors :)

you should write the `:after` and `:before` rules inside the previous block (which is `.react-datepicker-container`) as follows:
```scss
&:after, &:before {
  bottom: 100%;
  left: 50%;
  border: solid transparent;
  content: " ";
  height: 0;
  width: 0;
  position: absolute;
  pointer-events: none;
}

&:after {
  border-bottom-color: $picker-header-background;
  border-width: $picker-arrow-size;
  margin-left: -1 * $picker-arrow-size;
}

&:before {
  border-bottom-color: $picker-border;
  border-width: ($picker-arrow-size + 1);
  margin-left: -1 * ($picker-arrow-size + 1);
}
```